paths/accounts/index.yaml

```diff
@@ -41,4 +41,4 @@ x-code-samples:
   source: |-
     phrase accounts list \
     --access_token <token>
-
+x-cli-version: '2.5'
```

Comment:
has it been discussed with the orch team that we change to simply the version number without a prefix symbol?

Reply:
It depends what does this field mean. If it means that `2.5` is the minimal CLI version that support the endpoint fully, then it's OK.

Reply:
yes, that's what it would mean. I think, until we start deprecating some endpoints, that should cover the needs.